src/webgpu/capability_info.ts

```diff
@@ -117,20 +117,19 @@ export const kRegularTextureFormatInfo = /* prettier-ignore */ makeTable(
   'rgba32float':           [        true,              ,        ,        ,          ,      true,          ,          ,              16],
 } as const);
 /* prettier-ignore */
```

Comment:
[1] this should have moved up along with the header

Reply:
No need now.